00 Meta/Vault Infos/FAQ.md

```diff
@@ -1,25 +1,24 @@
 # FAQ
 
 
-> [!question] What is this about? What is Dataview?
+> [!question]- What is this about? What is Dataview?
 > Dataview is a very popular community plugin for [Obsidian.md](https://obsidian.md/). It lets you query your data inside your vault and can help you in various, powerful ways. Its potential is almost unlimited due to dataviews support for [javascript queries](https://blacksmithgu.github.io/obsidian-dataview/api/intro/)
 
-
 > [!question]- Who is the maintainer of this vault?
 > I'm Sam, a regular Obsidian user. You'll sometimes catch me in the dataview thread of the Obsidian Discord. I hope I won't stay the only maintainer for long - this was AB1908 idea in the first place, anyway.
 
 > [!question]- Why does this vault exists?
-> Discord is not the best place to store information for later access and while there exists an extensive Forum Thread for sharing Dataview queries, it is not super intuitive and user friendly to search in it. The idea is to instead everages Obsidians immense strength in storing data and make it accessible.
+> Discord is not the best place to store information for later access and while there exists an extensive Forum Thread for sharing Dataview queries, it is not super intuitive and user friendly to search in it. The idea is to instead leverages Obsidians immense strength in storing data, making it easily accessible and searchable and interconnect what belongs together so it's a breeze for you to find the dataview example you need.
```

Comment:
Typo "leverages" -> "leverage"
Grammar issue "parallel structures": `making it easily accessible and searchable and interconnect` doesn't really flow nicely.